rest/client/rest.go

```diff
@@ -126,6 +125,77 @@ func (c *communicatorImpl) GetHosts(ctx context.Context, f func([]*model.APIHost
 	return nil
 }
 
+func (c *communicatorImpl) SetBannerMessage(ctx context.Context, m string) error {
+	info := requestInfo{
+		method:  post,
+		version: apiVersion2,
+		path:    "admin/banner",
+	}
+
+	_, err := c.retryRequest(ctx, info, struct {
+		Banner string `json:"banner"`
+	}{
+		Banner: m,
+	})
+
+	return errors.Wrap(err, "problem setting banner")
+}
+
+func (c *communicatorImpl) GetBannerMessage(ctx context.Context) (string, error) {
+	info := requestInfo{
+		method:  get,
+		version: apiVersion2,
+		path:    "admin",
```

Comment:
admin/banner?

Reply:
admin/banner and admin/service_flags are PUT only, to get, use the /admin route.